src/fromager/wheels.py

```diff
@@ -189,13 +193,32 @@ def add_extra_metadata_to_wheels(
         for req_file in req_files:
             shutil.copy(req_file, dist_info_dir / f"fromager-{req_file.name}")
 
-        cmd = ["wheel", "pack", str(dist_info_dir.parent), "-d", str(wheel_file.parent)]
+        build_tag = ctx.settings.build_tag(req.name, version)
+
+        cmd = [
+            "wheel",
+            "pack",
+            str(dist_info_dir.parent),
```

Comment:
I think we want `os.fspath()` here instead of just `str()`.

Reply:
I am not sure would we need os.fspath? dist_info_dir is pathlib.Path object which has implemented the `__str__` method to return the string representation of the file system path

Reply:
Huh, we're using fspath in a bunch of other places, and I thought the reason was to ensure the right encoding. It doesn't look like that's what it does, though. @tiran why have we been using os.fspath?

Reply:
I think it is strictly pathlib.Path object